ouroboros-network/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -52,6 +52,7 @@ data NodeToNodeVersion
     -- ^ Changes:
     --
     -- * new 'KeepAlive' codec
+    -- * Enable @CardanoNodeToNodeVersion5@, i.e., Alonzo
```

Comment:
I'm wondering why this is not `NodeToNodeV_8`. Was it because the `NodeToNodeV_7` was not yet released?

Reply:
Yes; double-checking the versions was on my list for us to TODO today.

Reply:
Node 1.27.0 uses 9b279c7548ee549e1ed755cd1acb69b6e69d0c7b, which already includes `NodeToNodeV_7`.

However, on that commit, `NodeToNodeV_7` wasn't enabled in the `supportedNodeToNodeVersions` map for Cardano.

https://github.com/input-output-hk/ouroboros-network/blob/9b279c7548ee549e1ed755cd1acb69b6e69d0c7b/ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs#L315-L325

So I think it's still mutable. @coot @dcoutts confirm?

Reply:
@coot replied on Slack, confirming that the new KeepAlive is ready. So it's OK to also add Alonzo to NTN V_7 and then enable both Alonzo and KeepAlive2 at the same time 👍

Reply:
We also fixed-up our commit to set `latestReleasedNodeVersion` to NTN V6 and NTC V8 -- ie the ones that have been used by mainline releases. IE the predecessors of the versions that enable Alonzo (and as a coincidence enable KeepAlive2).

The resulting restriction of the node's default version negotation limit was the reason we added `latestReleasedNodeVersion` recently. And the corresponding new config file option `EnabledDevelopmentVersion` as of input-output-hk/cardano-node#2400 should let Benchmarking and Q&A etc _opt-in_ to negotiating the Alonzo-enabling versions.